crates/project_panel/src/project_panel.rs

```diff
@@ -2762,11 +2783,12 @@ impl ProjectPanel {
                 }
             }))
             .cursor_pointer()
+            .bg(bg_color)
             .child(
                 ListItem::new(entry_id.to_proto() as usize)
                     .indent_level(depth)
                     .indent_step_size(px(settings.indent_size))
-                    .selected(is_marked || is_active)
+                    .selectable(false)
```

Comment:
This was a tricky one. We had a `.hover` style adjustments in the div that contained the ListItem AND ListItem sets the background color as well if the ListItem is selectable. `selectable` does not change the behaviour of click events or anything else, it only controls the coloring.